dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -297,105 +255,78 @@ public static FuseUmountable launchFuse(FuseUmountable fuseFs,
   }
 
   /**
-   * Updates Alluxio configuration according to command line input.
+   * Converts command line options into Alluxio configuration.
    *
-   * @param cli  the command line inputs
-   * @param conf the modifiable configuration to update
+   * @param cli the command line inputs
+   * @return configuration containing the properties parsed from cli
    */
-  protected static void setConfigurationFromInput(CommandLine cli, InstancedConfiguration conf) {
-    if (cli.hasOption(MOUNT_POINT_OPTION_NAME)) {
-      conf.set(PropertyKey.FUSE_MOUNT_POINT,
-          cli.getOptionValue(MOUNT_POINT_OPTION_NAME), Source.RUNTIME);
-    }
-    if (cli.hasOption(MOUNT_ROOT_UFS_OPTION_NAME)) {
-      String ufs = cli.getOptionValue(MOUNT_ROOT_UFS_OPTION_NAME);
-      if (ufs.startsWith(Constants.SCHEME)) {
-        conf.set(PropertyKey.FUSE_MOUNT_ALLUXIO_PATH, ufs, Source.RUNTIME);
-      } else {
-        // Disable connections between FUSE and server
-        conf.set(PropertyKey.USER_METRICS_COLLECTION_ENABLED, false, Source.RUNTIME);
-        conf.set(PropertyKey.USER_UPDATE_FILE_ACCESSTIME_DISABLED, true, Source.RUNTIME);
-      }
-    }
-    if (cli.hasOption(MOUNT_OPTIONS_OPTION_NAME)) {
-      List<String> fuseOptions = new ArrayList<>();
-      String[] mountOptionsArray = cli.getOptionValues(MOUNT_OPTIONS_OPTION_NAME);
-      for (String opt : mountOptionsArray) {
-        String trimedOpt = opt.trim();
-        if (trimedOpt.isEmpty()) {
-          continue;
-        }
-        String[] optArray = trimedOpt.split("=");
-        if (optArray.length == 1) {
-          fuseOptions.add(trimedOpt);
-          continue;
-        }
-        String key = optArray[0];
-        String value = optArray[1];
-        if (PropertyKey.isValid(key)) {
+  protected static InstancedConfiguration parseCliOptionsAsConfig(FuseCliOptions cli) {
+    InstancedConfiguration conf = new InstancedConfiguration(new AlluxioProperties());
+    cli.getMountPoint()
+        .ifPresent(mp -> conf.set(PropertyKey.FUSE_MOUNT_POINT, mp, Source.RUNTIME));
+    cli.getUpdateCheck()
+        .ifPresent(updateCheckEnabled -> {
+          conf.set(FUSE_UPDATE_CHECK_ENABLED, updateCheckEnabled, Source.RUNTIME);
+        });
+    cli.getRootUfsUri()
+        .ifPresent(ufsRootUri -> {
+          conf.set(FuseOptions.FUSE_UFS_ROOT, ufsRootUri.toString(), Source.RUNTIME);
+          // Disable connections between FUSE and server
+          conf.set(PropertyKey.USER_METRICS_COLLECTION_ENABLED, false, Source.RUNTIME);
+          conf.set(PropertyKey.USER_UPDATE_FILE_ACCESSTIME_DISABLED, true, Source.RUNTIME);
+        });
```

Comment:
How to make sure the `MountPoint` and `RootUfsUri` are present?
They must exist. This is mandatory.

Reply:
if absent, the default values for `PropertyKey.FUSE_MOUNT_POINT` and `DORA_CLIENT_UFS_ROOT` will be used. which defaults to `/mnt/alluxio-fuse` and `<worker directory>/underFSStorage` respectively.